components/breadcrumb/index.css

```diff
@@ -34,6 +37,10 @@ governing permissions and limitations under the License.
   transform: logical scale(1) rotate(0deg);
 
   opacity: 1;
+
+  &.is-reversed {
```

Comment:
This new class is for supporting RTL, but I don't think we need it now. Should we remove it? See here for detail: [BreadcrumbItem.tsx#L59-L62](https://github.com/adobe-private/react-spectrum-v3/blob/f0f104dbd5e2e70b954e50308f64ec8b2e61ab7e/packages/%40react-spectrum/breadcrumbs/src/BreadcrumbItem.tsx#L59-L62)

Reply:
Yes, remove it, we don't need it. `transform: logical rotate(0deg)` above will flip it automatically.